contracts/mapping-token/contracts/darwinia/DarwiniaMappingTokenFactory.sol

```diff
@@ -88,14 +90,15 @@ contract DarwiniaMappingTokenFactory is Initializable, Ownable, DailyLimit {
         string memory symbol,
         uint8 decimals,
         address backing,
-        address source
+        address source,
```

Comment:
What's the meaning of `backing` and `source`?

Reply:
backing is the address of the backing model(backing contract address), and source is the origin erc20 token address.

Reply:
maybe it's better to call it backing_address and origin_address?

Reply:
How about `backing_contract` and `original_token`?

`original_token` means the token address on backing chain being mapped, and `mapping_token` means the mapping token generated on issuing chain, relative to `original_token`.

Reply:
`original_token` and `mapping_token` seems good.